numpy-tutorial/image_mod.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import matplotlib.image as mpimg
+
+img = mpimg.imread("kitten.png")
+print(type(img))
+print(img.shape)
+
+# => numpy.ndarray
```

Comment:
👩‍💻 **Technical Recommendation**

Since you're showing output in most of your Python source files, maybe it would make sense to replace them with Jupyter Notebooks (`.ipynb` files)? They allow to save the expected output with them. What do you think?

(You can take a look at [build-a-web-scraper/](https://github.com/realpython/materials/tree/master/build-a-web-scraper) folder, which comes with a few of those notebooks.)